src/olympia/scanners/tasks.py

```diff
@@ -229,14 +229,14 @@ def _delay_auto_approval_indefinitely(version):
         defaults={'auto_approval_delayed_until': datetime.max})
 
 
-def run_action(version_id):
+def run_action(version):
     """This function tries to find an action to execute for a given version,
     based on the scanner results and associated rules.
 
     It is not run as a Celery task but as a simple function, in the
     auto_approve CRON."""
-    log.info('Checking rules and actions for version %s.', version_id)
-    version = Version.objects.get(pk=version_id)
+    log.info('Checking rules and actions for version %s.', version.pk)
+    version = Version.objects.get(pk=version.pk)
```

Comment:
Do we really need to re-fetch the object then?

Reply:
oops, no we don't

Reply:
f3e63f18c308d6ed22b0cd12bda45d77e68fdebb